python/ray/scripts/scripts.py

```diff
@@ -876,10 +870,10 @@ def submit(cluster_config_file, docker, screen, tmux, stop, start,
 @click.argument("cluster_config_file", required=True, type=str)
 @click.argument("cmd", required=True, type=str)
 @click.option(
-    "--docker",
+    "--outside-docker",
```

Comment:
Instead of this, can you do something like:

```
--run-env="auto"
```
and then you can override that with different choices:
```
--run-env="auto"|"docker"|"host"
```
It's just a bit weird to have `--outside-docker=False` when the default IS outside-docker

Reply:
Oh--taht sounds good!